build/android/scripts/build_target.sh

```diff
@@ -73,27 +73,33 @@ export STRIP=${TOOL_PATH}-strip
 export CPPFLAGS="${CPPFLAGS} --sysroot=${SYSROOT} -I${SYSROOT}/usr/include -I${ANDROID_TOOLCHAIN}/include"
 export LDFLAGS="${LDFLAGS} -L${SYSROOT}/usr/lib${LIB_SUFFIX} -L${ANDROID_TOOLCHAIN}/lib -lc++_static -latomic -lm"
 
+BUILDDIR="${ROOTDIR}/jni/${DESTDIR_NAME}"
+
+export pkg_configure_flags="--host=${ARCH} --disable-shared --disable-asm"
+export pkg_make_flags=-j2
+export pkg_prefix=${BUILDDIR}
+
 (
-    cd $ROOTDIR
-    install -d ${ROOTDIR}/build/${ARCH}-${API}
-    cd ${ROOTDIR}/build/${ARCH}-${API}
+    cd "${ROOTDIR}/../../"
     test -f Makefile && make clean
     echo "Configure with --host=${ARCH} and toolchain ${ANDROID_TOOLCHAIN}"
     test -x ${ROOTDIR}/../../configure || (cd ${ROOTDIR}/../.. && ./autogen.sh)
+    ${ROOTDIR}/../../build/dependency all
```

Comment:
@AntonioLangiu Perhaps you could rewrite this line as:

```sh
${ROOTDIR}/../../build/dependency geoip
pkg_configure_flags="$pkg_configure_flags --disable-asm" ${ROOTDIR}/../../build/dependency libressl  # XXX
${ROOTDIR}/../../build/dependency libevent
```

This should pass the extra configure flag only to libressl and clearly highlight the temporary workaround.

What do you think? Would you like to try whether this approach works?